Classes/Core/RequestHandler.php

```diff
@@ -64,16 +64,18 @@ public function getPriority()
     public function handleRequest()
     {
         $request = ServerRequest::fromGlobals();
+        // TODO: Required for b/c - can be removed with Flow 7
         $response = new \GuzzleHttp\Psr7\Response();
         $this->componentContext = new ComponentContext($request, $response);
+        $this->bootstrap->setEarlyInstance(ComponentContext::class, $this->componentContext);
```

Comment:
Note: Those lines can be removed again with https://github.com/neos/flow-development-collection/pull/2221